src/engine/sys/sdl_glimp.cpp

```diff
@@ -653,17 +653,16 @@ static rserr_t GLimp_SetMode( int mode, bool fullscreen, bool noborder )
 	if ( SDL_GetDesktopDisplayMode( r_displayIndex->integer, &desktopMode ) == 0 )
 	{
 		displayAspect = ( float ) desktopMode.w / ( float ) desktopMode.h;
-
 		logger.Notice("Display aspect: %.3f", displayAspect );
 	}
 	else
 	{
 		Com_Memset( &desktopMode, 0, sizeof( SDL_DisplayMode ) );
 
-		logger.Notice("Cannot determine display aspect (%s), assuming 1.333", SDL_GetError() );
+		logger.Notice("Cannot determine display aspect, assuming 1.333: %s", SDL_GetError() );
```

Comment:
"Assuming 1.333" doesn't make sense because this number is never actually used anywhere. Maybe just remove the display aspect logs entirely?

Reply:
That `displayAspect` variable seems to be a global one, used there:

https://github.com/DaemonEngine/Daemon/blob/a971371ef5d1df8a7ece8961f4f3865046635aba/src/engine/sys/sdl_glimp.cpp#L377-L402

Though, it is set by default to `0.0f`:

https://github.com/DaemonEngine/Daemon/blob/a971371ef5d1df8a7ece8961f4f3865046635aba/src/engine/renderer/tr_init.cpp#L31

So maybe we have a bug to not set it to `1.333` when we say we assume it's `1.333`. Also maybe we want to assume something else than `4/3` in 2021. I don't know what's it's for anyway.

Reply:
I added log for dimension and “actually set it to 1.333”.